### ISSUE TITLE
Missing GHIssueCommentChanges

### ISSUE THREAD

**Issue by matusfaro** [hub4j/github-api#1243]
Missing `GHIssueCommentChanges` similar to [`GHIssueChanges`](https://github.com/hub4j/github-api/blob/main/src/main/java/org/kohsuke/github/GHIssueChanges.java) as per [`issues_comment`](https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#issue_comment) GitHub API.

Thanks!

**Comment by ReyhanPatria** [comment]
Hi! I'd like to work on this if you'd assign me to it @bitwiseman, thanks!

**Comment by willtsai** [comment]
Hello! Full disclosure here: I'm working on this issue as well for my Software Engineering course project, so even if my eventual code changes become redundant, I can at least provide additional perspective on implementing this feature.

Thanks,
-Will

**Comment by willtsai** [comment]
Hi @bitwiseman - I'm trying to use `mvn install -Dtest.github.takeSnapshot -Dtest.github.org=false -Dtest=YourTestClassName` per the contribution guide to run my unit test and take a json snapshot for the stub data so that I may convert it to a Wiremock test before I submit a PR. 

However, I have run `mvn install -Dtest.github.takeSnapshot -Dtest.github.org=false -Dtest=GHEventPayloadTest#issue_comment_edited` several times against the unit test below, but it's never been able to produce stub data stored in the `src/test/resources/org/kohsuhke/github/YourTestClassName/wiremock` path as described in the contributor guide. I'm running this on JDK 11, think that might be the issue? I haven't been able to get JDK 8 installed on my ARM64 Mac so have not tried this on Java 8.
```java
    @Test
    public void issue_comment_edited() throws Exception {
        GHIssueComment comment = getGitHubBuilder().withOAuthToken(System.getenv("GITHUB_OAUTH"))
                .build()
                .getRepository("willtsai/githubapi-testrepo")
                .getIssue(1)
                .getComments()
                .get(0);
        comment.update("comment one");
        String lastCommentBody = comment.getBody();
        comment.update("comment two");
        assertThat(comment.getBody(), is("comment two"));
    }
```

**Comment by bitwiseman** [comment]
Not sure why this would not work for you. I'll invite you to the hub4j test  org. It might be a bug in the non-test-org feature.

**Comment by willtsai** [comment]
ok thanks! I will try another unit test using `getRepository()` targeting the test org repo

**Comment by willtsai** [comment]
Update: that seems to have unblocked me, thank you @bitwiseman! Seems like there might indeed be a bug in the non-test-org feature. Should I log an issue for this?

**Comment by willtsai** [comment]
@bitwiseman - after looking at this more today, it seems that I didn't need to use Snapshot to generate the test data stubs after all given that this issue deals with `GHEventPayload` so I was able to get the event data json stub from a test webhook I added a test repo of mine. 

That being said, I did notice that the comment changes object in the event payloads look the same for both issue comments and PR comments (maybe other types of comments as well). 

**Thus, I'm wondering if you prefer that I add a generic `GHCommentChanges` wrapper that can then be used for all sorts of comments (`GHIssueComments`, GHPullRequestReviewComment`, etc.) or should I keep the wrapper specialized for each type of comment (e.g. `GHIssueCommentChanges`, `GHPullRequestReviewCommentChanges`, etc.) even though they will be identical?**

```json
  "changes": {
    "body": {
      "from": "pull request comment BEFORE edit"
    }
  },
```
